code/design/cloud/distrubuted-sys.md

```
@@ -0,0 +1,41 @@
+## What is a distrubuted system??
+
+**Primer**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)